src/rosetta-helpers.ts

```diff
@@ -88,6 +194,42 @@ function makeFeeOperation(tx: BaseTx): RosettaOperation {
   return fee;
 }
 
+function makeBurnOperation(tx: DbStxEvent, baseTx: BaseTx, index: number): RosettaOperation {
+  const burn: RosettaOperation = {
+    operation_identifier: { index: index },
+    type: getAssetEventTypeString(tx.asset_event_type_id),
+    status: getTxStatus(baseTx.status),
+    account: {
+      address: unwrapOptional(baseTx.sender_address, () => 'Unexpected nullish sender_address'),
+    },
+    amount: {
+      value: '-' + unwrapOptional(tx.amount, () => 'Unexpected nullish amount').toString(10),
```

Comment:
The `'-'` doesn't sit right with me. I have the same type of behavior where I need to check for the negative sign in amount. It feels hacky.